### ISSUE TITLE
Transmission Weight Not Scaling Correctly with Gear Ratios

### ISSUE THREAD

**Issue by A1ex-Andro** [Muushy/Sprocket-Feedback#567]
**Version:** 0.22

For a manual transmission, if you increase the first gear ratio and therefore the output torque in 1st gear, you are only increasing the diameter of that gear on the output shaft (while decreasing the size of the corresponding gear on the input shaft), and increasing the diameter of the output shaft to withstand the increased torque. That is all. The input shaft's diameter does not need to be increased, since the input torque does not change. The weight and size should not increase so significantly as it does in the game.

Note that by decreasing the top gear, you are decreasing the diameter of the gear on the output shaft, but increasing the diameter of the corresponding gear on the input shaft. The diameters of the shafts however, do not change in this case.

**In Summary:**

- The **input torque** of the engine determines the thickness of the **input shaft**.
- The maximum **output torque** determines the thickness of the **output shaft**.
- For a gear ratio of 2:1, the gears are the **same size** as for 0.5:1, only switched places (one gear has twice the diameter in either case).
- For **every** gear ratio, (diameter of output gear) + (diameter of input gear) = **the same**, because they are all on the same input and output shafts.

**Comment by A1ex-Andro** [comment]
It may be useful to watch this video showing how a manual transmission works:
[https://www.youtube.com/watch?v=wCu9W9xNwtI](url)
As well as this classic video:
[https://www.youtube.com/watch?v=JOLtS4VUcvQ](url)
The section of this video also has a good animation:
[https://www.youtube.com/watch?v=TUpeMYTj1O8&t=360s](url)

Once you have a good understanding of the basics, check out this demonstration:
[https://www.youtube.com/watch?v=vOo3TLgL0kM](url)

**Comment by Muushy** [comment]
Thanks A1ex Andro!
Think you could take a look at the new transmission scaling in 0.2.50? - It should be quite a bit more accurate now
